src/View/HelperRegistry.php

```diff
@@ -88,7 +88,8 @@ public function __isset(string $helper): bool
      * Provide public read access to the loaded objects
      *
      * @param string $name Name of property to read
-     * @return \Cake\View\Helper|null
+     * @return object|null
+     * @psalm-return \Cake\View\Helper|null
```

Comment:
We might be able to keep the more specific one here?

Reply:
phpstan throws error since the parent class has `object|null` but I guess we could add an ignore for it.